NSS-Day-Cohort-42/rare-API-media-moguls
# Rp header image

#### Changes Made
1. Changed Model for post to include upload_to directory in image field
1. Added code in create and update methods in the post.py view file to save images sent by the user
​
#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout rp-headerImage
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> It'll be easiest to test while also running my client side branch ```rp-headerImage```
>nuke database with ```./nuke.sh``` since changes were made to the post model
>try to create a new post and save an image with it
>try to edit a post and change the image
>you should also still be able to save posts without an image the image_url property is allowed to be null

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
